src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,56 @@
+package edu.kis.vh.nursery;
+
+/**
+ * Rhymer, contains array with max size 11
+ */
+public class DefaultCountingOutRhymer {
+	final int CAPACITY = 12;
+	final int EMPTY = -1;
+	private final int[] numbers = new int[CAPACITY];
+	public int total = -1;
+
+	/**
+	 * adds value to array numbers
+	 * @param in value to add
+	 */
+	protected void countIn(int in) {
```

Comment:
Visibility modifiers should be changed only for class fields. Classes without any visible methods and fields are useless.